aten/src/ATen/BatchedTensorImpl.cpp

```diff
@@ -29,6 +29,7 @@ BatchedTensorImpl::BatchedTensorImpl(Tensor value, BatchDims bdims)
     sizes_and_strides_.size_at_unchecked(dim) = value_sizes.at(actual_dim);
     sizes_and_strides_.stride_at_unchecked(dim) = value_strides.at(actual_dim);
   }
+  storage_offset_ = value_.storage_offset();
```

Comment:
Why is this the right value to use? Should we remove the potential offset from the batch dims?

Reply:
It is okay for each slice of the batched tensor to inherit the same offset, if that is what you mean (for example consider an op like slice[1:]). I don't think the potential offset from the batch dims matters in this particular case because these properties are for the logical tensor.